py3status/modules/keyboard_layout.py

```diff
@@ -3,9 +3,14 @@
 Display keyboard layout.
 
 Configuration parameters:
+    button_next: Button to click to cycle up through available layouts
+        (default 4)
+    button_prev: Button to click to cycle down through available layouts
```

Comment:
(Q) Maybe we avoid `up/down` here.
```
button_next: Button to click to cycle next available layout
button_prev: Button to click to cycle previous available layout
```
Based on other modules, I would do this.
```
button_next: mouse button to skip next layout (default 4)
button_previous: mouse button to skip previous layout (default 5)
```
---------------
New comment: I guess that's a no on... `(default [])` for `layouts`?
```
layouts: specify a list of layout to use (default [])
```

Reply:
The thing is I'm sort of loathed to use `[]` as a default this is because `def fn(default=[]):` is a really really bad thing to ever do in python.

 I think it is also bad to do here too.  I should probably look at making this less of a problem.  The reason is that it is the same list not a fresh one.